engine/installation/linux/docker-ce/ubuntu.md

```diff
@@ -0,0 +1,284 @@
+---
+description: Instructions for installing Docker CE on Ubuntu
+keywords: requirements, apt, installation, ubuntu, install, uninstall, upgrade, update
+redirect_from:
+- /engine/installation/ubuntulinux/
+- /installation/ubuntulinux/
+- /engine/installation/linux/ubuntulinux/
+title: Get Docker CE for Ubuntu
+---
+
+{% assign minor-version = "17.03" %}
+
+To get started with Docker CE on Ubuntu, make sure you
+[meet the prerequisites](#prerequisites), then
+[install Docker](#install-docker).
+
+## Prerequisites
+
+### Docker EE customers
+
+To install Docker Enterprise Edition (Docker EE), go to
+[Get Docker EE for Ubuntu](/engine/installation/linux/docker-ee/ubuntu/)
+**instead of this topic**.
+
+To learn more about Docker EE, see
+[Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
+
+### OS requirements
+
+To install Docker CE, you need the 64-bit version of one of these Ubuntu
+versions:
+
+- Yakkety 16.10
+- Xenial 16.04 (LTS)
+- Trusty 14.04 (LTS)
+
+Docker CE is supported on both `x86_64` and `armhf` architectures.
+
+### Uninstall old versions
+
+Older versions of Docker were called `docker` or `docker-engine`. If these are
+installed, uninstall them:
+
+```bash
+$ sudo apt-get remove docker docker-engine
```

Comment:
we may want to add `docker.io` here, as there is a package named like that in the ubuntu repositories https://packages.ubuntu.com/trusty/docker.io